### PR TITLE
Three Klipse related libraries

### PR DESCRIPTION
**Pull request by viebel**

### CODE REVIEW

**Comment by weavejester** [comment]
Can you replace the `-` with a `_` for the keys?

**Comment by weavejester** [comment]
Also, instead of the "Leiningen Plugins" category for the Codox theme, instead use "Codox Themes".

**Comment by viebel** [comment]
Done

File: projects.yml [hunk 1]
```diff
@@ -3957,3 +3957,21 @@ ezzmq:
...
+  platforms: [cljs]
+  
+klipse_plugin:
+  name: Klipse Web Plugin
+  url: https://github.com/viebel/klipse
+  categories: [Interactive Development]
+  platforms: [cljs]
+
+codox_klipse_theme:
+  name: Codox
```
Comment (weavejester) [inline projects.yml]: The name is "Codox" when it should be "Codox Klipse Theme"

**Comment by weavejester** [comment]
Thanks! I've just noticed the name of the `codox_klipse_theme` key is wrong, though.

**Comment by viebel** [comment]
Good catch! 
Fixed

**Pull request merged**